cfme/fixtures/v2v.py

```diff
@@ -57,25 +37,28 @@ def v2v_providers(request, second_provider, provider, v2v_downstream_enable):
     v2v_providers.rhv_provider.delete_if_exists(cancel=False)
 
 
-@pytest.fixture(scope='function')
+@pytest.fixture(scope="module")
 def host_creds(request, v2v_providers):
     """Add credentials to VMware and RHV hosts."""
     if len(v2v_providers) > 2:
-        pytest.skip("There are more than two providers in v2v_providers fixture,"
-                    "which is invalid, skipping.")
+        pytest.skip(
+            "There are more than two providers in v2v_providers fixture,"
+            "which is invalid, skipping."
+        )
     try:
         vmware_provider = v2v_providers.vmware_provider
         rhv_provider = v2v_providers.rhv_provider
         vmware_hosts = vmware_provider.hosts.all()
         for host in vmware_hosts:
-            host_data, = [data for data in vmware_provider.data['hosts']
-                          if data['name'] == host.name]
-            host.update_credentials_rest(credentials=host_data['credentials'])
+            host_data, = [
+                data for data in vmware_provider.data["hosts"] if data["name"] == host.name
+            ]
+            host.update_credentials_rest(credentials=host_data["credentials"])
 
         rhv_hosts = rhv_provider.hosts.all()
         for host in rhv_hosts:
-            host_data, = [data for data in rhv_provider.data['hosts'] if data['name'] == host.name]
-            host.update_credentials_rest(credentials=host_data['credentials'])
+            host_data, = [data for data in rhv_provider.data["hosts"] if data["name"] == host.name]
```

Comment:
Please wrap this in a `try` for the `ValueError` that this syntax raises. Skip the test, using the exception message since it describes why the generated list is invalid.

```
In [1]: d, = []
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-1-f2af38263c16> in <module>()
----> 1 d, = []

ValueError: need more than 0 values to unpack

In [2]: d, = [1]

In [3]: d, = [1, 2]
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-3-0349359da3b3> in <module>()
----> 1 d, = [1, 2]

ValueError: too many values to unpack

```